test/ui/test_stitch_axis.py

```diff
@@ -0,0 +1,84 @@
+import os
+import pytest
+from RefRed.main import MainGui
+from qtpy import QtCore, QtWidgets
+
+
+wait = 100
+
+
+def test_stitch_plot(qtbot, data_server):
+    """Test the axis adjustment in stitching plot"""
+    # Create the main window.
+    main = MainGui()
```

Comment:
Can you use fixture [main_gui](https://github.com/neutrons/RefRed/blob/next/test/conftest.py#L80) instead of lines 13 to 33? Plus you won't need fixture `data_server`

Reply:
Is there a way for me to specify a different xml file using `main_gui`?

Reply:
`main_gui(configuration=data_server.path_to("some_conf.xml"), show=False)`

Reply:
thanks, I will update the test now